src/effects/builtin/whitenoiseeffect.cpp

```diff
@@ -0,0 +1,76 @@
+#include "effects/builtin/whitenoiseeffect.h"
+
+#include "util/rampingvalue.h"
+
+namespace {
+const QString dryWetParameterId = QStringLiteral("dry_wet");
+} // anonymous namespace
+
+// static
+QString WhiteNoiseEffect::getId() {
+    return QStringLiteral("org.mixxx.effects.whitenoise");
+}
+
+// static
+EffectManifestPointer WhiteNoiseEffect::getManifest() {
+    EffectManifestPointer pManifest(new EffectManifest());
+    pManifest->setId(getId());
+    pManifest->setName(QObject::tr("White Noise"));
+    pManifest->setAuthor("The Mixxx Team");
+    pManifest->setVersion("1.0");
+    pManifest->setDescription(QObject::tr("Creates active white noise sound"));
```

Comment:
```suggestion
    pManifest->setDescription(QObject::tr("Creates active white noise sound"));
    pManifest->setEffectRampsFromDry(true);
```
The ramping code in the effect takes care of this if `previous_drywet` starts at 0 when the effect is loaded and disabled/enabled.